project-keeper/src/main/java/com/exasol/projectkeeper/validators/pom/PomFileIO.java

```diff
@@ -66,18 +66,17 @@ private void writePomFile(final Document document, final OutputStream outputStre
         }
     }
 
-    /**
-     * Read a pom file as XML document.
-     * 
-     * @param pomFile pom file
-     * @return xml document
+    /*
+     * Inspired by https://stackoverflow.com/a/33564346
```

Comment:
This is public now. Maybe add a real comment.
```suggestion
     * Trim the whitespace from the given node and its children.
     *
     * Inspired by https://stackoverflow.com/a/33564346
```